README.md

```diff
@@ -12,6 +12,87 @@ Invoke the Makefile:
 make build
 ```
 
+## Configuration
+
+Configuration is done by JSON file. The json file should describe a complete topology by describing various services,
+their routes, and the downstreamCalls that those routes make. `rootRoutes` indicate the root spans or ingress points
+ into the topology, and can be configured with the number of traces per hour to send.
+
+Different tags can also be added to each service or to each route. Tags added to a service will apply to all routes for
+that service, whereas tags for a route will be added for that route only. The tags for a route supersede tags set for
+a service. Services and routes can set tags probabilistically based on integer weights (default is 1). Additionally,
```

Comment:
The usage below doesn't make it clear how to use it: it seems percentage in the places that it replaced the error generation (e.g.: 15 + 85 weights). A clarification here seems good.

Reply:
Updated to describe the exact prob The (15+85) imo just makes the math easy to immediately see how likely each is (vs doing the reduced 3+17).